velox/functions/sparksql/specialforms/SparkCastHooks.cpp

```diff
@@ -50,6 +50,15 @@ Expected<Timestamp> SparkCastHooks::castIntToTimestamp(int64_t seconds) const {
   return Timestamp(seconds, 0);
 }
 
+Expected<Timestamp> SparkCastHooks::castDoubleToTimestamp(double value) const {
+  if (FOLLY_UNLIKELY(std::isnan(value) || std::isinf(value))) {
```

Comment:
Drop FOLLY_UNLIKELY, it is only used in some rarely case even not happen. Nan check is not such case.